utonics/labproject/main.go

```diff
@@ -325,6 +325,7 @@ func newProject(values map[string][]string, botClient, userClient *worker.Client
 		return msgs, err
 	}
 	for smName := range submodules {
+	  smName = strings.ReplaceAll(smName, "/", "_")
```

Comment:
Something's wrong with the indentation here.  Is this indented with spaces?
You can use `gofmt` to automatically format code.

Reply:
I hope gofmt worked.